backend/fetchers/base.rb

```diff
@@ -36,7 +36,7 @@ def get_filer(row)
       if filer_id.nil?
         filer_id = 0
       elsif (/^\d*$/ =~ filer_id).nil?
-        puts row.values_at('filer_namf', 'filer_naml').join(' ') + " invalid id:" + filer_id;
+        puts row.values_at('filer_namf', 'filer_naml').join(' ') + " invalid id:" + filer_id.to_s;
```

Comment:
Okay. You can avoid this by doing string interpolation instead of concatenation, e.g.:

```ruby
"invalid id: #{filer_id}"
```

the #{} implicitly calls #to_s on whatever's inside.